src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -348,6 +351,14 @@ void UserInfoPopup::updateUserData()
 
         this->userId_ = id;
 
+        // this can be nullptr if the setting is not enabled.
+        if (this->ui_.userIDLabel)

```

Comment:
shouldn't have to check this now since the setting is removed

Reply:
done